docs/integrations/kubeflow.md

```diff
@@ -0,0 +1,91 @@
+
+    ```python
+     import kfp
+     from kfp import dsl
+     from kubernetes.client.models import V1EnvVar
+     
+     def create_branch_op():
+        return dsl.ContainerOp(
+        name='create_branch',
+        image='treeverse/lakectl',
+        arguments=['branch', 'create', 'lakefs://example-repo/example-branch', '-s', 'lakefs://example-repo/main']).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_ACCESS_KEY_ID',value='AKIAIOSFODNN7EXAMPLE')).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY',value='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY')).add_env_variable(V1EnvVar(name='LAKECTL_SERVER_ENDPOINT_URL',value='https://lakefs.example.com'))
+     ```
+2. Commit changes to a branch: A ContainerOp that commits uncommitted changes to `example-branch` on `example-repo`.
+
+     ```python
+     def commit_op():
+         return dsl.ContainerOp(
+         name='commit',
+         image='treeverse/lakectl',
+         arguments=['commit', 'lakefs://example-repo/example-branch', '-m', 'commit message']).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_ACCESS_KEY_ID',value='AKIAIOSFODNN7EXAMPLE')).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY',value='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY')).add_env_variable(V1EnvVar(name='LAKECTL_SERVER_ENDPOINT_URL',value='https://lakefs.example.com'))
+     ```
+3. Merge two lakeFS branches: A ContainerOp that merges `example-branch` into the `main` branch of `example-repo`.
+    
+    ```python
+     def merge_op():
+         return dsl.ContainerOp(
+         name='merge',
+         image='treeverse/lakectl',
+         arguments=['merge', 'lakefs://example-repo/example-branch', 'lakefs://example-repo/main']).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_ACCESS_KEY_ID',value='AKIAIOSFODNN7EXAMPLE')).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY',value='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY')).add_env_variable(V1EnvVar(name='LAKECTL_SERVER_ENDPOINT_URL',value='https://lakefs.example.com'))
+     ```
+
+You can invoke any lakeFS operation supported by `lakectl` by implementing it as a ContainerOp. Check out the complete [CLI reference](https://docs.lakefs.io/reference/commands.html) for the list of supported operations. \ 
+*Note:*  Some `lakectl` operations may require mounting data sources into the pod running their corresponding ContainerOps. For example, the `lakectl fs` operations.   
+
+## Add the lakeFS steps to your pipeline
+Add the steps created on the previous step to your pipeline before compiling it. 
+
+### Example pipeline
+{: .no_toc }
+A pipeline that implements a simple ETL, that has steps for branch creation and commits.    
+
+```python
+def lakectl_pipeline():
+   create_branch_task = create_branch_op()
+   extract_task = example_extract_op() 
+   commit_task = commit_op() 
+   transform_task = example_transform_op()
+   commit_task = commit_op()
+   load_task = example_load_op()
+```
+
+**Note**  \
```

Comment:
why do you force line break using '\' - from over notes we have, we don't force split here

Reply:
It looked nicer to me, but I changed it now to follow the convention.